### DISCUSSION TITLE
Introduction: Alex Goodall

### DISCUSSION THREAD

**Discussion by Alex-Goodall** [life-itself/community#198] [Introductions]
Hello Life Itself Community

I recently had a chat with Nathan Fitchen (thanks, Nathan), and he invited me join in here. Delighted to be invited.

_My core interest at the moment_ 
**The Relevant Education Project**
 _"What you need to know to consciously participate in the most consequential 25 years in Humanity’s history"_

[https://relevant.education/](https://relevant.education/)

It has (will have) many strands, including (of course...) a book.

_My background_
Started with maths and physics, worked in IT, early AI, knowledge management, digital marketing and, currently, private tutoring.

_My engagement with meaning_ 
It started pretty early on in life, encompassing Christianity, a couple of Indian Gurus, a touch of sorcery (via Carlos Castaneda and his gang), a period of rational atheism (which took me by complete surprise), re-enchantment via Ken Wilber and Integral Philosophy, and most recently a deep dive into Analytic Idealism – which in my view is the most disruptive and consequential metaphysical theory.

_Alongside my deeper meaning_ path:
There’s been a spiral of opening up that included Metamodernism, GameB and the huge ecosystem of what I call the **_Humanity’s Transition_** community which has been flourishing more explicitly since around 2019: typified by this very community and the work of its participants.

_How I spend my time_
Teaching maths, science and computer science as a private tutor – which I love – and carving out time to spend on my projects related to Humanity’s transition: 
**The Liminal Learning Portal** [(http://tllp.org/)](http://tllp.org/) (development is on hold)
and, as mentioned above, primarily
**Relevant Education**

**Comment by nathenf** [comment]
Hey @Alex-Goodall,

Glad you made it! We have a bunch of ongoing discussions here that you can get involved in, I would suggest having notifications on for discussions to stay updated. 

@rufuspollock @theo-cox just tagging you for visibility

**Comment by nathenf** [comment]
@Alex-Goodall this in particular might be of interest https://github.com/life-itself/community/discussions/186

**Comment by rufuspollock** [comment]
@Alex-Goodall a very warm welcome! Amazing projects!
